state/accumulator.go

```diff
@@ -22,12 +23,13 @@ import (
 // Accumulate function for timeline events. v2 sync must be called with a large enough timeline.limit
 // for this to work!
 type Accumulator struct {
-	db            *sqlx.DB
-	roomsTable    *RoomsTable
-	eventsTable   *EventTable
-	snapshotTable *SnapshotTable
-	spacesTable   *SpacesTable
-	entityName    string
+	db              *sqlx.DB
+	roomsTable      *RoomsTable
+	eventsTable     *EventTable
+	snapshotTable   *SnapshotTable
+	spacesTable     *SpacesTable
+	entityName      string
+	snapshotSizeVec *prometheus.HistogramVec // TODO: Remove, this is temporary to get a feeling how often a new snapshot is created
```

Comment:
maybe `snapshotMemberCountVec`, since it only counts #members and not the total number of state events?